.github/workflows/testing.yml

```diff
@@ -128,3 +127,8 @@ jobs:
           interop/interop_test.sh
           echo -e "\n-- Running xDS E2E Test --"
           xds/internal/test/e2e/run.sh
+          echo -e "\n-- Running protoc-gen-go-grpc change detector test --"
+          ./vet.sh -install
+          (cd test/tools && go install google.golang.org/protobuf/cmd/protoc-gen-go)
+          (cd cmd/protoc-gen-go-grpc && go install .)
```

Comment:
This kind of thing should be in `protoc-gen-go-grpc_test.sh`.

Reply:
Done